source/common/config/utility.h

```diff
@@ -238,6 +247,14 @@ class Utility {
   static std::vector<Stats::TagExtractorPtr>
   createTagExtractors(const envoy::api::v2::Bootstrap& bootstrap);
 
+  /**
+   * Creates the set of tags if TagSpecifiers have "fixed_value" field. The
+   * tags will use as default tags. In other words, the tags will be added into
+   * all metrics.
+   * @param bootstrap bootstrap proto.
+   */
+  static std::vector<Stats::Tag> createTags(const envoy::api::v2::Bootstrap& bootstrap);
```

Comment:
Should this be internal to utility.cc and just live in the anonymous namespace?

Reply:
I'm not familiar but since this function needs to be called from `server/server.cc`, I thought this function needs to be published. Is that wrong?

Possibly will be refactored with https://github.com/envoyproxy/envoy/pull/2357#discussion_r161341362.

Reply:
Refactoring with e0596988a490ae5a0ad78271523d2cc9046d6896 not to expose these functions. Good point!